app/vmagent/remotewrite/remotewrite.go

```diff
@@ -121,6 +121,9 @@ var (
 	// rwctxsDefault contains statically populated entries when -remoteWrite.url is specified.
 	rwctxsDefault []*remoteWriteCtx
 
+	// labelsCompressor contains information about all compressed labels for streaming aggregation
+	labelsCompressor promutils.LabelsCompressor
```

Comment:
Can we make this as global var in `lib/streamaggr`, and define `vm_streamaggr_labels_compressor_size_bytes ` and `vm_streamaggr_labels_compressor_items_count ` there as well?

Reply:
already made this changes, but just pushed them now